embedded-redis/src/main/java/com/playtika/test/redis/RedisProperties.java

```diff
@@ -33,9 +33,9 @@
 @ConfigurationProperties("embedded.redis")
 public class RedisProperties extends CommonContainerProperties {
     static final String BEAN_NAME_EMBEDDED_REDIS = "embeddedRedis";
-    String dockerImage = "redis:4.0.12";
-    String user = "root";
-    String password = "passw";
-    String host = "localhost";
-    int port = 6379;
+    public String dockerImage = "redis:4.0.12";
```

Comment:
why this become public?

Reply:
RedisProperties is used inside wait strategies which moved into "wait" subpackage